Wikipedia/assets/mediawiki-extensions-CodeMirror/resources/lib/codemirror/lib/codemirror.js

```diff
@@ -9017,10 +9017,10 @@ ContentEditableInput.prototype.setUneditable = function (node) {
 };
 
 ContentEditableInput.prototype.onKeyPress = function (e) {
```

Comment:
@joewalsh 
Will probably want to fork the mediawiki code mirror bits ( https://github.com/wikimedia/wikipedia-ios/pull/2808/files#diff-60bb710c5144ead8f8cb73a22de579abR13 ) or this and the next commit will get overwritten when we run carthage update.